src/sqliteformat.h

```diff
@@ -408,7 +408,7 @@ class MKCAL_EXPORT SqliteFormat
 #define SELECT_COMPONENTS_BY_PLAIN \
 "select * from Components where DateStart=0 and DateEndDue=0 and DateDeleted=0"
 #define SELECT_COMPONENTS_BY_RECURSIVE \
-"select * from components where ((ComponentId in (select DISTINCT ComponentId from recursive)) or (RecurId!=0)) and DateDeleted=0"
+"select * from components where ((ComponentId in (select DISTINCT ComponentId from recursive)) or (ComponentId in (select DISTINCT ComponentId from rdates)) or (RecurId!=0)) and DateDeleted=0"
```

Comment:
Old detail, but this and the next one seem slightly inconsistent with other queries by not capitalizing the table names.